From: Ben Hutchings <ben@decadent.org.uk>
Subject: net: define netdev_features_t
Date: Sun, 27 Jan 2013 23:59:43 +0000
Bug-Debian: http://bugs.debian.org/699129
Forwarded: not-needed

This was added by commit c8f44affb7244f2ac3e703cab13d55ede27621bb upstream.

---
--- a/include/linux/netdevice.h
+++ b/include/linux/netdevice.h
@@ -57,6 +57,8 @@ struct netpoll_info;
 struct phy_device;
 /* 802.11 specific */
 struct wireless_dev;
+
+typedef u32 netdev_features_t;
 					/* source back-compat hooks */
 #define SET_ETHTOOL_OPS(netdev,ops) \
 	( (netdev)->ethtool_ops = (ops) )
